test/integration/backward_compatible/sql/ExecuteTest.js

```diff
@@ -499,6 +499,10 @@ describe('SqlExecuteTest', function () {
             client = await Client.newHazelcastClient({
                 clusterName: cluster.id
             });
+            // There was a fix regarding originatingMemberId in 5.0 and 4.2.x after 4.2.
+            // TODO: Update here to be 4.2.1 if we release 4.2.1
+            // https://github.com/hazelcast/hazelcast-nodejs-client/pull/940
+            TestUtil.markClientVersionAtLeast(this, '5.0.0');
```

Comment:
Just a thought, what do you think about not skipping this test for 5.0- clients, but change the assertion according to the client version? So for 4.2 client, we could use member.uuid, for 5.0 we could use client uuid?

Reply:
I thought of doing that, but I think the correct uuid is client uuid, isn't it? But at the same time, I think assertion of originatingMemberId is just a single part of this test and we should run the other parts for 4.2 clients.

In total, I think we can change the assertion as you suggested

Reply:
Yes, maybe don't check the originatingMemberUuid on 5.0- clients, but do other assertions on all versions

Reply:
Sorry, I didn't saw your last commit, I am fine with different assertions on different versions

Reply:
Ok. Let's go with the current version in 78b0c3f